store/tikv/sql_fail_test.go

```diff
@@ -69,3 +79,39 @@ func (s *testSQLSuite) TestFailBusyServerCop(c *C) {
 
 	wg.Wait()
 }
+
+func (s *testSQLSuite) TestCoprocessorStreamRecvTimeout(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("create table t (id int)")
+	for i := 0; i < 200; i++ {
+		tk.MustExec(fmt.Sprintf("insert into t values (%d)", i))
+	}
+
+	// rowsPerChunk in MockTiKV is 64, so the result will be 4 chunks.
+	enable := true
+	ctx := context.WithValue(context.Background(), "mockTiKVStreamRecvHook", func(ctx context.Context) {
+		if !enable {
+			return
+		}
+
+		select {
+		case <-ctx.Done():
+		case <-time.After(time.Minute):
```

Comment:
It takes 20s, because I set 20s constant for `Recv()` timeout.
I should have use  the timeout in `SendReq()`, but maybe it's better to do it in another PR